src/main/java/org/csanchez/jenkins/plugins/kubernetes/volumes/workspace/EmptyDirWorkspaceVolume.java

```diff
@@ -46,9 +49,12 @@ public class EmptyDirWorkspaceVolume extends WorkspaceVolume {
     @CheckForNull
     private Boolean memory;
 
+    private String requestsSize;
+
     @DataBoundConstructor
-    public EmptyDirWorkspaceVolume(Boolean memory) {
+    public EmptyDirWorkspaceVolume(Boolean memory, String requestsSize) {
         this.memory = memory;
+        this.requestsSize = requestsSize;
     }
```

Comment:
Since you also added a setter annotated with `@DataBoundSetter`, you don't need to update the constructor.